src/opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1293,6 +1293,7 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
         const std::string prefix = "| "; /* logger prefix string */
         this->snapshots.resize(reportStep + 1);
         auto& input_block = this->m_sched_deck[reportStep];
+        std::unordered_map<std::string, double> wellpi_global_factor;
```

Comment:
It is much appreciated if you can change the name `wellpi_global_factor` to be `wpimult_global_factor`. It should have been done in the previous PR while it got merged first.

Reply:
Good point, did not notice but will change it.